JS/Validate_Contact_form.js

```diff
@@ -0,0 +1,15 @@
+/* The below code is selecting the form element from the DOM. */
+const form = document.querySelector('form');
+const emailInput = document.getElementById('user_email');
+
+form.addEventListener('submit', (event) => {
+  /* The below code is a regular expression that checks if the email is valid. */
+  const emailRegex = /[A-Z]/;
+  // Amin writes this code
+  if (emailRegex.test(emailInput.value)) {
+    document.getElementById('email_message').classList.toggle('disabled');
+    event.preventDefault();
+  } else if (!document.getElementById('email_message').className === 'email-message disabled') {
+    document.getElementById('email_message').classList.toggle('disabled');
+  }
```

Comment:
- [ ] As the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md) suggest, you should try as much as possible to ensure that the email input is case-sensitive. if a user submits the contact form with the email field containing only uppercase characters the error message can show up around the input fields or just around the submit button ☺️.

Reply:
Hi  @AlphaNtihinduka , Thank you for your comment.
 Please can you make it clear your comment because unable to understand it.
This is  what done before
![image](https://user-images.githubusercontent.com/21217148/231520423-7846915a-ff71-4b3e-bde0-715f4e115175.png)